vg/vgsvg/vgsvg.go

```diff
@@ -9,12 +9,13 @@ package vgsvg
 import (
 	"bufio"
 	"bytes"
-	"code.google.com/p/plotinum/vg"
 	"fmt"
-	svgo "github.com/ajstarks/svgo"
 	"image/color"
 	"io"
 	"math"
+
+	svgo "github.com/ajstarks/svgo"
```

Comment:
I prefer svgo since that's the last element of the import path and well-behaved go packages use the last element name as their package name. I consider it a bug that the package name isn't actually svgo.